src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -73,7 +73,7 @@ class CreateProject extends React.Component {
       unitId: parseInt(this.props.organization.id, 10),
       param: 'exposed',
     });
-    trackAnalyticsEvent({
+    trackAdhocEvent({
       eventKey: 'new_project.visited',
       eventName: 'New Project Page Visited',
```

Comment:
the `eventName` is no longer needed

Reply:
Thanks, removed.

Reply:
I would have thought typescript would catch that, no?

Reply:
is it since `trackAdhocEvent` is in getsentry. we don't have type checking on the hook?